api/geth_backend.go

```diff
@@ -874,6 +874,22 @@ func (b *GethStatusBackend) SendTransaction(sendArgs transactions.SendTxArgs, pa
 	return
 }
 
+func (b *GethStatusBackend) SendTransactionWithChainID(chainID uint64, sendArgs transactions.SendTxArgs, password string) (hash types.Hash, err error) {
+	verifiedAccount, err := b.getVerifiedWalletAccount(sendArgs.From.String(), password)
+	if err != nil {
+		return hash, err
+	}
+
+	hash, err = b.transactor.SendTransactionWithChainID(chainID, sendArgs, verifiedAccount)
+	if err != nil {
+		return
+	}
+
+	go b.statusNode.RPCFiltersService().TriggerTransactionSentToUpstreamEvent(hash)
+
+	return
```

Comment:
@alaibe is this needed here?

Reply:
Will double and remove if not needed

Reply:
It's needed when you use named return parameters